### DISCUSSION TITLE
如何购买?

### DISCUSSION THREAD

**Discussion by laowang96** [getlantern/lantern-forum-cn#1073] [Q&A]
没有可用的支付方式

**Comment by MaxMKMax** [comment]
@laowang96 您好. 请您提供我您的账号邮箱. 我给您获取支付链接.

**Comment by MaxMKMax** [comment]
@laowang96 您好. 请您提供我您的账号邮箱. 我给您获取支付链接. 谢谢您支持蓝灯!

https://lantern.io/zh-Hans/faq
这个连接也可以帮到您.
